packages/clerk-js/src/utils/url.ts

```diff
@@ -292,6 +292,10 @@ export const generateSrc = ({ src, width }: { src?: string; width: number }) =>
     return '';
   }
 
+  if (src.substring(0, 5) === 'data:') {
```

Comment:
❓ why not `if (src.startsWith('data:')) {` ?

Reply:
Right, :D